src/lib.rs

```diff
@@ -34,6 +34,7 @@ pub use crate::shims::intrinsics::EvalContextExt as IntrinsicsEvalContextExt;
 pub use crate::shims::tls::{EvalContextExt as TlsEvalContextExt, TlsData};
 pub use crate::shims::dlsym::{Dlsym, EvalContextExt as DlsymEvalContextExt};
 pub use crate::shims::env::{EnvVars, EvalContextExt as EnvEvalContextExt};
+pub use crate::shims::io::{FileHandler, EvalContextExt as FileEvalContextExt};
```

Comment:
`io` is extremely generic... since this is all about files and file descriptors, `file` seems like a better name to me.

Reply:
Or maybe `fs`, to match the libstd module name.

Reply:
https://github.com/rust-lang/miri/pull/983/commits/976c976f0948ebfb080f51c733c5bbe24c366554